vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/VGSCollect.kt

```diff
@@ -675,40 +714,28 @@ class VGSCollect {
         fun setEnvironment(env: String): Builder = this.apply { environment = env }
 
         /** Sets the VGSCollect instance to use the custom hostname. */
-        fun setHostname(cname: String): Builder {
-            if (cname.isURLValid()) {
-                host = cname.toHost()
-
-                if (host != cname) VGSCollectLogger.debug(message = "Hostname will be normalized to the $host")
-
-            } else {
+        fun setHostname(cname: String): Builder = this.apply {
+            if (!cname.isURLValid()) {
                 VGSCollectLogger.warn(message = context.getString(R.string.error_custom_host_wrong_short))
+                return@apply
             }
-
-            return this
+            this.host = cname
         }
 
+        /**
+         * Sets the VGSCollect instance to use the custom hostname port.
+         * Port cn be used only with localhost for ex. with VGS-Satellite, otherwise it will be ignored.
```

Comment:
Port can be used only with localhost with VGS-Satellite, otherwise, it will be ignored.